File: listaExercicios/Metodos_Estaticos/Estatico.java
class Estatico {
    public static long fatorial(int n) {
        if (n < 0) {
            throw new IllegalArgumentException("Fatorial não é definido para números negativos.");
        }
        if (n == 0 || n == 1) {
            return 1;
        }
        return n * fatorial(n - 1);
    }

    public static void main(String[] args) {
        System.out.println(fatorial(5));  // Saída: 120
    }
}

File: listaExercicios/Metodos_Estaticos/estatico.py
class Matematica:
    @staticmethod
    def fatorial(n):
        if n < 0:
            raise ValueError("Fatorial não é definido para números negativos.")
        if n == 0 or n == 1:
            return 1
        return n * Matematica.fatorial(n - 1)


print(Matematica.fatorial(5)) 
